staging/vhost-device-gpu/src/vhu_gpu.rs

```diff
@@ -595,35 +620,36 @@ impl VhostUserBackendMut for VhostUserGpuBackend {
         vrings: &[VringRwLock],
         _thread_id: usize,
     ) -> IoResult<()> {
+        // We use thread_local here because it is the easiest way to handle VirtioGpu being !Send
+        thread_local! {
+            static VIRTIO_GPU_REF: RefCell<Option<VirtioGpu>> = None.into();
```

Comment:
one small comment: Instead of `None.into()`, i think you should initialize it with `RefCell::new(None)`